From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Shelley Vohr <shelley.vohr@gmail.com>
Date: Mon, 1 Jun 2020 15:30:47 -0800
Subject: fix: export zlib symbols

This patch sets ZLIB_DLL so that we properly export zlib symbols.

diff --git a/third_party/zlib/BUILD.gn b/third_party/zlib/BUILD.gn
index a3378e43ac99e611925bd28b3d78f3683d4d57cc..4113be75214f864560efab5da88e51f7887654ef 100644
--- a/third_party/zlib/BUILD.gn
+++ b/third_party/zlib/BUILD.gn
@@ -312,6 +312,10 @@ component("zlib") {
   defines = []
   deps = []
 
+  if (is_win) {
+    defines += [ "ZLIB_DLL" ]
+  }
+
   if (!use_x86_x64_optimizations && !use_arm_neon_optimizations) {
     # Apparently android_cronet bot builds with NEON disabled and
     # we also should disable optimizations for iOS@x86 (a.k.a. simulator).
